src/aggregate.rs

```diff
@@ -73,6 +73,8 @@ pub enum Aggregation {
     Individual(AstarteType),
     /// Object data, also called aggregate. Can only be from a datastream.
     Object(HashMap<String, AstarteType>),
+    /// Unset of a property
+    Unset,
```

Comment:
Adding `Unset` under the `Aggregation enum` is a bit strange, I think  we can rename it in `AstartePayload` or 
move Aggregation in another enum that contains `Unset` and `Aggregation`, so :
1.
``` rust
 enum AstartePayload{
   Individual(AstarteType),
   Object(HashMap<String, AstarteType>),
   Unset,
 }
``` 
 2. 
 ``` rust
enum Aggregation{
   Individual(AstarteType),
   Object(HashMap<String, AstarteType>),
 }
 
 enum AstartePaylod {
  Aggregation, 
  Unset,
 }
 ```

Reply:
I think the first option whould be more ergonomic to use